.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
           - einops>=0.6.0
           - kornia>=0.6.9
           - lightning>=2.0.9
-          - matplotlib>=3.8.1
+          - matplotlib>=3.8.1,<3.9.1
```

Comment:
Note that this will be fixed once https://github.com/matplotlib/matplotlib/pull/28518 is merged